templates/compte/notifications_disabled.html

```diff
@@ -0,0 +1,16 @@
+{% extends "editorial/base.html" %}
+{% load crispy_forms_filters %}
+
+{% load static %}
+
+{% block page_title %}Désactivations des rappels{% endblock %}
+{% block robots %}noindex, nofollow{% endblock %}
+
+{% block editorial_content %}
+  <main role="main" id="content">
+    <h3 class="mb-3">Désabonnement</h3>
+    <div class="alert alert-info">
+      Votre demande a bien été prise en compte.
```

Comment:
I'd rather say *Vos préférences ont bien été mises à jour*, so we can have a generic, reusable `preferences_updated.html` template for future other preferences.

Also, why didn't you use flash messages?

Reply:
Well, the task was asking this, but thats makes more sens especially with your next remark